x/airdrop/genesis.go

```diff
@@ -0,0 +1,21 @@
+package airdrop
+
+import (
+	"github.com/ingenuity-build/quicksilver/x/airdrop/keeper"
+	"github.com/ingenuity-build/quicksilver/x/airdrop/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetParams(ctx, genState.Params)
+}
+
+// ExportGenesis returns the capability module's exported genesis.
+func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
+	return &types.GenesisState{
+		Params: k.GetParams(ctx),
+	}
```

Comment:
Iterate over all unclaimed claimRecords and export
Iterate over all zoneRecords and export